go-base/Makefile

```diff
@@ -23,6 +23,8 @@ GORUMP=$(TOP)/gorump
 GOBASE=$(TOP)/go-base
 GOBOOTSTRAP=$(TOP)/gobootstrap
 export GOROOT=$(GOBOOTSTRAP)
+unexport GOPATH
+unexport GOBIN
```

Comment:
Should this be  `unset`? or is `unexport` also standard? I couldn't find `unexport` locally on OSX.